src/pages/algorithms/sorting-algorithms/merge-sort/index.md

```diff
@@ -3,13 +3,30 @@ title: Merge Sort
 ---
 ## Merge Sort
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/algorithms/sorting-algorithms/merge-sort/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Merge Sort is a <a href='https://github.com/naveenbanda/guides/tree/master/src/pages/algorithms/divide-and-conquer-algorithms' target='_blank' rel='nofollow'>Divide and Conquer</a> algorithm. It divides input array in two halves, calls itself for the two halves and then merges the two sorted halves.
```

Comment:
Please change the link to `https://guide.freecodecamp.org/algorithms/divide-and-conquer-algorithms`